### PR TITLE
Python API integration

### PR DESCRIPTION
**Pull request by siahuat0727**
Hi team, this branch is about the Python API to use `nnfusion.jit`. I believe it should be ready to be reviewed and merged too. :) @wenxcs

### CODE REVIEW

File: src/python/nnfusion/executor.py [hunk 1]
```diff
@@ -80,7 +82,7 @@ class Executor(object):
         5: ("", ""),  # UNKNOWN
     }
 
-    def __init__(self, nnf_rt_dir):
+    def __init__(self, nnf_rt_dir, device=None):
         """
         Parameters:
             nnf_rt_dir: A full string path to nnfusion runtime,
```
Comment (mzmssg) [inline src/python/nnfusion/executor.py:88]: Could you add docstring for added parameters? And it would be great if there is docstring in runtime.py

Comment (siahuat0727) [reply src/python/nnfusion/executor.py:88]: No problem!

**Review by wenxcs — APPROVED** [review]

**Pull request merged**